packages/eslint-plugin-template/src/utils/create-eslint-rule.ts

```diff
@@ -25,12 +25,11 @@ export function getTemplateParserServices(context: any): ParserServices {
  * If @angular-eslint/template-parser is not the configured parser when the function is invoked it will throw
  */
 export function ensureTemplateParser(
-  context: TSESLint.RuleContext<string, []>,
+  context: TSESLint.RuleContext<string, readonly unknown[]>,
```

Comment:
I had to change the signature of `ensureTemplateParser` in order to compile:

![Captura de Tela 2021-01-04 às 02 20 24](https://user-images.githubusercontent.com/11965907/103503571-6f261f00-4e33-11eb-81a2-441f2911113c.png)

Reply:
Cool